README.md

```diff
@@ -30,25 +30,25 @@ This gem is an alternative and inspired by the great work in the other gem [simp
 Whereever you are integrating `SimpleCov` you can configure the `SimpleCovSmallBadge` gem as any formater can be configured. The default integration could looks as follows:
 
 ```ruby
-require 'simple-cov-small-badge'
+require 'simplecov_small_badge'
 
 # Wherever your SimpleCov.start block is (spec_helper.rb, test_helper.rb, or .simplecov)
 SimpleCov.start do
   # add your normal SimpleCov configs
   add_filter "/app/model"
   # call SimpleCov::Formatter::BadgeFormatter after the normal HTMLFormatter
-  SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+  SimpleCov.formatters = SimpleCov::Formatter::MultiFormatter.new([
     SimpleCov::Formatter::HTMLFormatter,
-    SimpleCovSimpleBadge::Formatter
-  ]
+    SimpleCovSmallBadge::Formatter
+  ])
 end
 
 # configure any options you want for SimpleCov::Formatter::BadgeFormatter
 SimpleCovSmallBadge.configure do |config|
   # does not created rounded borders
   config.rounded_border = true
   # set the background for the title to darkgrey
-  config.background_color = '#ffffcc'
+  config.background = '#ffffcc'
```

Comment:
According to the [configuration class](https://github.com/skyfusion89/simplecov-small-badge/blob/772269f2335bda7d1e9fa42ab96275c7e91dd121/lib/simplecov_small_badge/configuration.rb#L12) there is no **background_color** option.